app/gui/src/model/execution_context.rs

```diff
@@ -271,6 +273,43 @@ pub struct AttachedVisualization {
 
 
 
+// ======================
+// === ComponentGroup ===
+// ======================
+
+/// A named group of components (language elements displayed by the Component Browser).
+///
+/// Components are displayed in the Commponent Browser in groups, and such groups can be defined by
+/// libraries imported into the current execution context. To learn more about component groups,
+/// see the [Component Browser Design
+/// Document](https://github.com/enso-org/design/blob/e6cffec2dd6d16688164f04a4ef0d9dff998c3e7/epics/component-browser/design.md).
```

Comment:
I don't really understand why this is defined in `gui::model::execution_context`. It doesnt look to me like anything connected with an execution context.